### PR TITLE
Update /security/fips copy

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Updated /security/fips according to changes in [copy doc](https://docs.google.com/document/d/1moTJCAtFXKD7ZXrxB-EB7GPVyZRTSIdkP-0BhPZl69M/edit#)

## QA

- View the demo in your web browser at: https://ubuntu-com-11022.demos.haus/security/fips
- See that the page includes the changes listed in the copy doc

## Issue / Card

Fixes [#4680](https://github.com/canonical-web-and-design/web-squad/issues/4680)

### CODE REVIEW

**Review by minkyngkm — COMMENTED** [review]
Looks good @sowasred2012! I just left some comments on the copydoc as I found some texts were different but it was not highlighted.

File: templates/security/fips.html [hunk 1]
```diff
@@ -234,8 +228,8 @@ <h2 id="modules">Certified packages under FIPS 140</h2>
...
+    <h2>FIPS packages and security updates</h2>
+    <p>Each FIPS 140 certificate is valid for 5 years, however vulnerabilities happen, and it is our intention to publish fixed packages quickly, irrespective of their certification status. We therefore provide two alternative options. An option to remain with the certified cryptographic packages (called the 'fips' option), and an option to use the certified packages but include security fixes (called the 'fips-updates' option) when available. Check <a class="p-link--external" href="https://ubuntu.com/security/certifications/docs/fips-enablement">our documentation pages on how to enable these options</a>.</p>
```
[the remark this answers is not in the document]
Comment (sowasred2012) [reply templates/security/fips.html:232]: Good spot!

**Review by minkyngkm — APPROVED** [review]
I will give `+1 with changes` so you can merge it when it's ready. Please double check with the PM and update the texts I commented on the copy doc if needed. Thanks @sowasred2012 :)

**Comment by minkyngkm** [comment]
LGTM @sowasred2012! 👍

**Pull request merged**